main.go

```diff
@@ -17,21 +17,20 @@ package main
 import (
 	"context"
 	"flag"
-	"fmt"
+	"github.com/caddyserver/certmagic"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"github.com/minio/minio-go/v7/pkg/s3utils"
 	"io/ioutil"
 	"log"
 	"net"
 	"net/http"
 	"net/url"
 	"os"
+	"path"
 	"strconv"
 	"strings"
 	"time"
-
-	"github.com/caddyserver/certmagic"
-	minio "github.com/minio/minio-go/v7"
-	"github.com/minio/minio-go/v7/pkg/credentials"
-	"github.com/minio/minio-go/v7/pkg/s3utils"
```

Comment:
Please use `goimports -w` to fix the import - this is unnecessary movement of import paths.